.travis.yml

```diff
@@ -14,7 +14,7 @@ notifications:
 
 env:
   - EMBER_TRY_SCENARIO=default
-  - EMBER_TRY_SCENARIO=ember-1.13
+  - EMBER_TRY_SCENARIO=ember-release
```

Comment:
We should continue to ensure we maintain backwards compatibility for some reasonable period.  I would keep 1.13 as a scenario and add ember-release